### DISCUSSION TITLE
How to mute tab in seleniumbase?

### DISCUSSION THREAD

**Discussion by SSujitX** [seleniumbase/SeleniumBase#2971] [General]
**How can i mute and unmute any tab in seleniumbase?**

**Comment by mdmintz** [comment]
See the StackOverflow post on sound control: https://stackoverflow.com/a/39392601/7058266
You can pass `--mute-audio` via `chromium_arg`:
`chromium_arg="--mute-audio"` / `--chromium-arg="--mute-audio"`

**Reply by SSujitX** [reply]
Thank you. 

`with SB(uc=True, chromium_arg="--mute-audio") as sb:`

It's working fine for me. 

One more thing, how can I add multiple args in chromium_arg?

**Reply by mdmintz** [reply]
For multiple args, use a comma-separated list with no spaces.
Eg: `chromium_arg="--arg1,--arg2"`.
